geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/api/CacheOperationMessageMarker.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.distributed.internal.membership.api;
+
+/**
+ * This interface in implemented by message classes performing cache
+ * operations and is used to check for message type internally.
+ */
+public interface CacheOperationMessageMarker {
```

Comment:
I think we should pull this checking out of membership. Even if we leave this marker in, the javadocs above don't really describe what this is doing or why someone would add this marker to a message.

Reply:
I'd be happy with pulling this out, maybe on a separate ticket...?

Reply:
We can do this as a separate ticket if you prefer.

Reply:
https://issues.apache.org/jira/browse/GEODE-7695